Kauri/3box-comments-tutorial.md

```diff
@@ -0,0 +1,333 @@
+
+Adding these new functions:
+
+```js
+// src/App.js
+...
+
+class App extends React.Component {
+  ...
+
+  componentDidMount() {
+    this.login();
+  }
+
+  async login() {
+    // Request access to wallet and pull addresses.
+    const addresses = await window.ethereum.enable();
+    const address = addresses[0];
+
+    // Create the 3Box instance.
+    const box = await Box.openBox(address, window.ethereum, {});
+    const profile = await Box.getProfile(address);
+
+    // Update the box in our state once we've synced.
+    box.onSyncDone(() => {
+      this.setState({
+        box: box
+      });
+    });
+
+    // Update our state immediately with current values.
+    this.setState({
+      box: box,
+      profile: profile,
+      address: address,
+      ready: true,
+    });
+  }
+
+  ...
+}
+
+...
+```
+
+### Creating the Component
+
+We're finally ready to create our component. We're going to update our [`render()`](https://reactjs.org/docs/react-component.html#render) function so that it renders the component we imported from `3box-comments-react`. Once we're finished with this step, we'll have have a working comment section!
+
+The Comments component takes a few required parameters. These parameters are:
+
+| Property          | Type     | Description                                                                                                                        |
+|-------------------|----------|------------------------------------------------------------------------------------------------------------------------------------|
+| `spaceName`       | `string` | Some [3Box Space](https://docs.3box.io/api/storage) name for your application. You'll usually want to use a single `spaceName` throughout your entire application. |
+| `threadName`      | `string` | Some [3Box Thread](https://docs.3box.io/api/messaging) name for this specific component. Each component typically has its own `threadName`.                          |
+| `adminEthAddr`    | `string` | An Ethereum address for the admin of your application. Admins are able to moderate comments in the comment thread.                 |
+| `box`             | `Object` | A `3box` instance. In our case, the one we initialized and added to our `state`.                                                   |
+| `currentUserAddr` | `string` | The address of the user viewing the component. We pulled this from the `ethereum` instance earlier.
+
+Let's go ahead and create the component using the values we initialized in `login()`. Make sure to replace `spaceName`, `threadName`, and `adminEthAddr` with your own values:
+
+```js
+// src/App.js
+...
+
+class App extends React.Component {
+  ...
+
+  render() {
```

Comment:
@kfichter So should this replace the current `render`?

Reply:
Yes - I'll go ahead and clarify that in the text. Thank you!